websockets/blob.go

```diff
@@ -0,0 +1,68 @@
+package websockets
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/grafana/sobek"
+
+	"go.k6.io/k6/js/common"
+)
+
+type blob struct {
+	typ  string
+	data bytes.Buffer
+}
+
+func (b *blob) text() string {
+	return b.data.String()
+}
+
+func (r *WebSocketsAPI) blob(call sobek.ConstructorCall) *sobek.Object {
```

Comment:
Wasn't this supposed to be exported? :thinking:  https://github.com/grafana/xk6-websockets/pull/74#discussion_r1644691748

Reply:
Ah, I thought that [being part of module `Exports()`](https://github.com/grafana/xk6-websockets/blob/9e992116dc47b0dbb6f3724231dc87bcc689347e/websockets/websockets.go#L48) is enough.
Wdyt? cc/ @mstoykov

Reply:
Ah, probably true, it's likely my misunderstanding :+1:

Reply:
Don't worry, in any case that made me realize that it's probably good idea to also add a couple tests to ensure that the `Blob`'s basic functionality works as expected (https://github.com/grafana/xk6-websockets/pull/74/commits/89d1b52c4359dc55e0d3a6cc295c1256e1ac9b7e) 👌🏻